apps/root/src/hooks/useAvailablePairs.ts

```diff
@@ -10,7 +10,7 @@ function useAvailablePairs(chainId: number) {
     'getAvailablePairs'
   );
 
-  return availablePairs[chainId];
+  return availablePairs[chainId] || [];
```

Comment:
This was an error occurring on 
apps/root/src/common/components/token-picker-modal/components/token-picker/index.tsx trying to read 'reduce' from undefined (line 439)

Reply:
oh thanks